src/problem_depot/problems/sdp.jl

```diff
@@ -1361,3 +1361,35 @@ end
         end
     end
 end
+
+@add_problem sdp function sdp_min_maxeig_canon_lmi(handle_problem!, ::Val{test}, atol, rtol, ::Type{T}) where {T, test}
+    # Minimize the maximum eigenvalue of an affine matrix function of a vector argument. Formulated
+    # as a linear optimization subject to a canonical LMI (SDP) constraint:
+    #
+    # minimize      λ
+    # subject to    A(x)⪯λ
```

Comment:
```suggestion
    # subject to    A(x) ⪯ λ*I
```

Reply:
Sure. Mathematically it is certainly more appropriate.